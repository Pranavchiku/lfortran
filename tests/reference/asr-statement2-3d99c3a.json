{
    "basename": "asr-statement2-3d99c3a",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/statement2.f90",
    "infile_hash": "f4f733c4e3689505cd43316198c495623ab88c5817a565ec643a610d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-statement2-3d99c3a.stdout",
    "stdout_hash": "d5b0195b8f38149ced8e431c1eb01bf7c1ecadf1d060e0784a502886",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}