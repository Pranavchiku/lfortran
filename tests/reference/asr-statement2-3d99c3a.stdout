(TranslationUnit 
    (SymbolTable
        1
        {
            errjac:
                (Function 
                    (SymbolTable
                        2
                        {
                            dfloat:
                                (Variable 
                                    2 
                                    dfloat 
                                    ReturnVar 
                                    () 
                                    () 
                                    Default 
                                    (Real 8 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            dp:
                                (Variable 
                                    2 
                                    dp 
                                    Local 
                                    (FunctionCall 
                                        2 kind 
                                        () 
                                        [((RealConstant 
                                            0.000000 
                                            (Real 8 [])
                                        ))] 
                                        (Integer 4 []) 
                                        (IntegerConstant 8 (Integer 4 [])) 
                                        ()
                                    ) 
                                    (IntegerConstant 8 (Integer 4 [])) 
                                    Parameter 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            i:
                                (Variable 
                                    2 
                                    i 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            j:
                                (Variable 
                                    2 
                                    j 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            kind:
                                (ExternalSymbol 
                                    2 
                                    kind 
                                    4 kind 
                                    lfortran_intrinsic_kind 
                                    [] 
                                    kind 
                                    Private
                                ), 
                            sum:
                                (Variable 
                                    2 
                                    sum 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Real 8 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            test:
                                (Variable 
                                    2 
                                    test 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Real 8 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            ti:
                                (Variable 
                                    2 
                                    ti 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Real 8 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                )
                            
                        }) 
                    errjac 
                    [] 
                    [(Return)] 
                    () 
                    Source 
                    Public 
                    Implementation 
                    () 
                    .false. 
                    .false. 
                    .false. 
                    .false. 
                    .false. 
                    [] 
                    [] 
                    .false.
                ), 
            lfortran_intrinsic_kind:
                (IntrinsicModule lfortran_intrinsic_kind)
            
        }) 
    []
)
